### DISCUSSION TITLE
Using GTK-server with Objeck

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#297] [General]
While our GTK windowing is still not usable, I think we could use GTK-server instead. This is GTK-server:

https://www.gtk-server.org/

The problem is that I don't know how to use it with Objeck.

@objeck Could you play with it a bit and give me some tutorials? Sample codes would be awesome. Thank you.

**Comment by objeck** [comment]
Thanks, I will take a look at this once I get my development  box up and running. The last part should show up tonight however, I need to do a complete rebuild. Not sure how long that’s going to take.

**Comment by ghost** [comment]
It seems we are looking at the wrong solution and GTK-server is not what we need at all.
